examples/mycoind/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Assumes you have everything set up
+# and have initialised the chain
+
+tendermint node --home ~/.mycoind --p2p.skip_upnp > ~/.mycoind/tendermint.log &
```

Comment:
Simple enough.
Since I am one for shell magic, I would make the directory (`~/.mycoind`) configurable somehow, but yeah, it guess it is nicer like this if you just want to run them.

The docs can also import this code literally.